File: 3477/solution.py
class Solution:
    def numOfUnplacedFruits(self, fruits: List[int], baskets: List[int]) -> int:
        res = 0
        basket_count = len(baskets)
        for fruit in fruits:
            is_not_set = 1
            for i in range(basket_count):
                if fruit <= baskets[i]:
                    is_not_set = 0
                    baskets[i] = 0
                    break
            res += is_not_set
        return res

File: 3477/Solution.java
class Solution {
    public int numOfUnplacedFruits(int[] fruits, int[] baskets) {
        int res = 0;
        int basket_count = baskets.length;
        for (int fruit : fruits){
            int is_not_set = 1;
            for (int i = 0; i < basket_count; i++){
                if (fruit <= baskets[i]){
                    baskets[i] = 0;
                    is_not_set = 0;
                    break;
                }
            }
            res = res + is_not_set;
        }
        return res;
    }
}